etstool.py

```diff
@@ -462,6 +464,30 @@ def test_all(edm):
         sys.exit(1)
 
 
+@cli.command()
+@edm_option
+@click.option('--runtime', default="3.6")
+@click.option('--toolkit', default="pyqt5")
+@click.option(
+    "--environment", default=None, help="Name of EDM environment to check."
+)
+@click.option(
+    '--strict/--not-strict',
+    default=False,
+    help="Use strict configuration for flake8 [default: --not-strict]",
+)
+def flake8(edm, runtime, toolkit, environment, strict):
+    """Run a flake8 check in a given environment."""
+    parameters = get_parameters(edm, runtime, toolkit, environment)
+    config = ""
+    if strict:
+        config = "--config=flake8_strict.cfg "
```

Comment:
I'm not sure what this file is. should this be `setup.cfg`?

Also, I don't fully understand what the point of "strict" vs "non-strict" options for flake8 is - don't we always want to use "strict"?

Reply:
This is a copy-pasta from TraitsUI - the idea was strict is something like what we are aiming for (and so new code should pass strict), but there are a lot of issues that need fixing before old code could pass the strict version.

Reply:
Obviously I'll fix one way or another.